src/software/ai/hl/stp/action/chip_action.h

```diff
@@ -15,7 +15,7 @@ class ChipAction : public Action
     /**
      * Creates a new ChipAction
      */
-    explicit ChipAction();
+    explicit ChipAction() = delete;
```

Comment:
this constructor is being used, so please don't delete it.

Reply:
should explicit KickAction(); be deleted? Since it also seems to be the only constructor similar to `ChipAction`. @jonathanlew

Reply:
no it should not